### PR TITLE
Fixed avatar volume adjustment

### PR DESCRIPTION
**Pull request by ksuprynowicz**
Fixes https://github.com/overte-org/overte/issues/839

### CODE REVIEW

File: assignment-client/src/audio/AudioMixerClientData.cpp [hunk 1]
```diff
@@ -223,12 +223,24 @@ void AudioMixerClientData::parseInjectorGainSet(ReceivedMessage& message, const
...
+        return;
+    }
+
+    // If someone is not speaking at the moment, then stream is not in active streams, and volume adjustment will fail,
+    // so we need to search in inactive streams too
+    auto itInactive = std::find_if(_streams.inactive.cbegin(), _streams.inactive.cend(), [nodeID](const MixableStream& mixableStream){
+        return mixableStream.nodeStreamID.nodeID == nodeID && mixableStream.nodeStreamID.streamID.isNull();
+    });
+
+    if (itInactive != _streams.active.cend()) {
```
Comment (HifiExperiments) [inline assignment-client/src/audio/AudioMixerClientData.cpp:241]:
```suggestion
    if (itInactive != _streams.inactive.cend()) {
```

these 2 blocks are pretty similar, could also consider moving them to a function that takes in the list of streams (active or inactive)

File: libraries/audio/src/AudioHRTF.h [hunk 2]
```diff
@@ -80,7 +82,10 @@ class AudioHRTF {
     //
     // HRTF local gain adjustment in amplitude (1.0 == unity)
     //
-    void setGainAdjustment(float gain) { _gainAdjust = HRTF_GAIN * gain; };
+    void setGainAdjustment(float gain) {
+        _gainAdjust = HRTF_GAIN * gain;
+        qDebug() << "AudioHRTF::setGainAdjustment: " << _gainAdjust;
```
Comment (HifiExperiments) [inline libraries/audio/src/AudioHRTF.h:87]: maybe remove before merging so we don't clutter the logs?

**Review by HifiExperiments — CHANGES_REQUESTED** [review]

File: libraries/audio/src/AudioHRTF.h (hunk 2 above)
Comment (ksuprynowicz) [reply libraries/audio/src/AudioHRTF.h]: Thanks for spotting this!
I removed it

File: assignment-client/src/audio/AudioMixerClientData.cpp (hunk 1 above)
Comment (ksuprynowicz) [reply assignment-client/src/audio/AudioMixerClientData.cpp]: Thanks! I made it into a function and it indeed looks much cleaner this way

**Review by HifiExperiments — APPROVED** [review]

**Pull request merged**